src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -71,16 +75,26 @@ def invalid_metric_tags(tags: Mapping[str, str]) -> Sequence[str]:
     return invalid_strs
 
 
+# TODO: Move this to where we do use case registration
```

Comment:
One question, what's the reason for wanting to move this?

Reply:
This was a [suggestion](https://github.com/getsentry/sentry/pull/47113#discussion_r1166434786) left by Nikhar.
The general reasoning is that we want to have centralized the definition of what a Use Case is, and it should be in that use case registration file.